robottelo/cli/factory.py

```diff
@@ -83,10 +84,10 @@ def create_object(cli_object, options, values):
 
     """
     update_dictionary(options, values)
-    result = cli_object.create(options)
-
-    # If the object is not created, raise exception, stop the show.
-    if result.return_code != 0:
+    try:
+        result = cli_object.create(options)
+    except CLICommandExecutionError:
```

Comment:
I don't know too much about the details of the CLI test suite, but in general I'd prefer small changes over large changes.